person.rb

```diff
@@ -1,15 +1,20 @@
 require_relative 'corrector'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
 
 class Person
```

Comment:
- [ ] Your **`Person`** class should have a method for adding rentals. Kindly add this. Suggestion: The method can take in two parameters (date and book) and returns an instance of the **Rental** class that takes in `date`, `self` and `book` as parameters